### ISSUE TITLE
LabBuilder.py: TypeError: replace() argument 2 must be str, not tuple

### ISSUE THREAD

**Issue by tsluyter** [DefensiveOrigins/APT-Lab-Terraform#6]
Just a small hint, should people run into the following error:

```
$ python3 LabBuilder.py -m ${MY_IP}
Traceback (most recent call last):
  File "LabBuilder.py", line 194, in <module>
    main()
  File "LabBuilder.py", line 191, in main
    buildmain(mgmtip)
  File "LabBuilder.py", line 152, in buildmain
    staticinfo=staticinfo.replace('clid',client_id)
TypeError: replace() argument 2 must be str, not tuple
```

This issue occurs when you're copy-pasting the output from "_az ad sp create-for-rbac_" directly into LabBuilder.py. It means that you left a comma at the end of the relevant field. For example:

`client_id       = "ddd-ccc-bbb-aaa-xxx",`

versus:

`client_id       = "ddd-ccc-bbb-aaa-xxx"`

**Comment by tsluyter** [comment]
Can be closed when pull req #9 is applied.

**Comment by tsluyter** [comment]
Tackled by completed merge.
